src/request-api.js

```diff
@@ -33,24 +34,18 @@ function onRes (buffer, cb) {
 
     if (chunkedObjects) {
       const parsed = []
-      res.on('data', chunk => parsed.push(JSON.parse(chunk)))
+      res.on('data', chunk => {
+        try {
+          parsed.push(JSON.parse(chunk))
+        } catch (err) {
+          // Browser quirks emit more than needed sometimes
```

Comment:
What I'm thinking is that "stream" and "chunked" should not exist at the same time. "chunked" is a stream made of small parts of the same files, they should not be parsed individually. "stream" in the another hand, it is the ndjson stuff :)